### PR TITLE
Add nano pytorch training bf16 example

### PR DESCRIPTION
**Pull request by y199387**
## Description
Add Nano PyTorch BFloat16 example

### 1. Why the change?
Add Nano fundamental example

### 2. User API changes
N/A

### 3. Summary of the change 
N/A

### 4. How to test?
- [x] Unit test

### CODE REVIEW

File: python/nano/tutorial/training/pytorch/pytorch_train_bf16.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                output = model(data)
+                loss = loss_fuc(output, target)
+                self.backward(loss)
+                optimizer.step()
+
+                train_loss += loss.sum()
+                num += 1
+            print(f'Train Epoch: {epoch}, avg_loss: {train_loss / num}')
+
+        # TESTING LOOP
```
Comment (TheaperDeng) [inline python/nano/tutorial/training/pytorch/pytorch_train_bf16.py:99]: are we adapting some external code to this tutorial? if not, we may ignore the testing loop here.

Comment (y199387) [reply python/nano/tutorial/training/pytorch/pytorch_train_bf16.py]: Done

**Review by TheaperDeng — APPROVED** [review]
LGTM

File: python/nano/tutorial/training/pytorch/pytorch_train_bf16.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+import torch
+from torch import nn
+from torch.utils.data import DataLoader
+from torchvision import transforms
+from torchvision.models import resnet18
+from torchvision.datasets import OxfordIIITPet
+from torchmetrics import Accuracy
+
+from bigdl.nano.pytorch.torch_nano import TorchNano
```
Comment (jason-dai) [inline python/nano/tutorial/training/pytorch/pytorch_train_bf16.py:24]: change to `from bigdl.nano.pytorch import TorchNano` @MeouSker77 @Oscilloscope98

**Pull request merged**